seqr/views/apis/staff_api_tests.py

```diff
@@ -380,3 +640,31 @@ def test_success_story(self):
 
         self.assertEqual(len(response_json['rows']), 1)
         self.assertDictEqual(response_json['rows'][0], EXPECTED_SUCCESS_STORY)
+
+    @mock.patch('seqr.views.apis.staff_api.export_multiple_files')
+    @responses.activate
+    def test_anvil_export(self, mock_export_multiple_files):
+        url = reverse(anvil_export, args=[PROJECT_GUID])
```

Comment:
Are you planning on doing a separate pr for the sample_metadata_export or do you want to do it as part of this PR?

Reply:
My plan is to do it in the same PR after the anvil_export is confirmed to be good. But I can change it a separated PR if you see any issues.

Reply:
I probably do the sample_metadata_export next week with another PR.